libraries/physics/src/MultiSphereShape.cpp

```diff
@@ -486,7 +486,10 @@ void MultiSphereShape::calculateDebugLines() {
                     break;
                 }
             }
-        }        
+        }
+        if (radiuses.size() == 0) {
+            radiuses.push_back(0.0f);
```

Comment:
interesting…is there a way we could instead detect lower down in the callstack (in calculateChamferBox, or lower) that radiuses (radii??) is empty?  I worry someone else will call it and crash

Reply:
`radiuses` is a local array variable of `calculateDebugLines()` with a complex calculation as to whether items should be added to it. 
The crash occurs further down the call stack in `calculateSphereLines(...)` if the array is empty, when item `[0]` is retrieved.

To handle an empty `radiuses` further down the call stack would be a matter of working through the code to see how to avoid calculating and displaying a sphere, and handle possible assumptions that there is in fact a sphere to display.

I could either add a "FIXME" comment to the proposed change, or pursue things further down the stack sometime "soon".

Reply:
can we just early exit out of calculateChamferBox at the top if radiuses.size() == 0?  does that crash somewhere else in the code?

Reply:
That alternative change also fixes the crash, though I don't know what side-effects not doing the chamfer box calculations may have.

Reply:
can we ever really know the side-effects of doing or not doing things